environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java

```diff
@@ -38,12 +49,22 @@ public static Builder builder() {
 
     public static class Builder {
 
+        private static final String TYPE_CIDR_WIDE_OPEN = "CIDR_WIDE_OPEN";
```

Comment:
smells like it should have been an enum

Reply:
They are converted to String anyway, I would rather delete the static declarations

Reply:
done